### ISSUE TITLE
Small issues with palette viewer.

### ISSUE THREAD

**Issue by AyanamiRei1** [finalburnneo/FBNeo#610]
Hello, I've been using FB Neo for a little while but there's something that always bothered me and it's that the palette viewer it's weird with CPS1/2 games it's backwards like in the wrong order and also it doesn't give the end user proper ARGB values just hex which really isn't useful for finding palettes within games.

This is how the palette viewer currently is.
![image](https://user-images.githubusercontent.com/25233407/100791620-66e05b80-3411-11eb-997d-009baf42808f.png)

and this what it really should be.
![image](https://user-images.githubusercontent.com/25233407/100791946-ce96a680-3411-11eb-88bc-12e48959e49f.png)

So basically what I'm asking could these be fixed if possible.
 
Oh and as for the version it's always been like this since the Final Burn Alpha days.

Thank you.

**Comment by barbudreadmon** [comment]
wondering if it could be related to https://github.com/finalburnneo/FBNeo/issues/605

**Comment by dinkc64** [comment]
It's not the palette viewer's fault :) 
Our CPS1/2 emulation was at heart, programmed from the ground up "differently", I'm not sure why our elders did it like this, but there probably is a good reason.
I'll look deeper into this and see what I can find out..

p.s. can you tell me a CPS1/2 game which uses something besides 0xff for 'A' in ARGB?

best regards,
- dink

**Comment by AyanamiRei1** [comment]
Basically all the street fighter games use 0 as A from WW to Alpha 3 they all use 0 as Alpha.

**Comment by dinkc64** [comment]
Thanks,
I just submitted a fix to display the CPS1/2 palettes in the correct order.   There isn't much I can do about the "A" value.  RGB are definitely there, though.

best regards,
- dink
